internal/mock/postgresql/postgresql.go

```diff
@@ -5,43 +5,46 @@
 package mocks
 
 import (
+	reflect "reflect"
+
 	gomock "github.com/golang/mock/gomock"
 	postgresql "github.com/sorintlab/stolon/internal/postgresql"
-	reflect "reflect"
 )
 
-// MockPGManager is a mock of PGManager interface
+// MockPGManager is a mock of PGManager interface.
```

Comment:
if we're going to make formatting changes like this in our fork, let's do it as a separate commit, and add it to `.git-blame-ignore-revs` so that PRs with functional changes have only the functional changes and not unrelated formatting changes